src/lib.rs

```diff
@@ -351,6 +350,32 @@ impl<T: Translation> Mapping<T> {
         )
     }
 
+    /// Invalidate `range` in the TLBs, so that permission changes are guaranteed to have taken
```

Comment:
Nit: Mostly I've been following Google style for doc comments, where the comment starts with a phrase that could follow "This method...". So this should be 'Invalidates' rather than 'Invalidate'.